common/djangoapps/student/models.py

```diff
@@ -981,10 +981,24 @@ def enrollment_counts(self, course_id):
         Returns a dictionary that stores the total enrollment count for a course, as well as the
         enrollment count for each individual mode.
         """
+        exclude_kwargs = {
```

Comment:
These same `kwargs` are also being defined in this file in the other PR, #831 .

Instead of duplicating them, we should share a constant between the two methods.

```python
# Top-level constant
EXCLUDE_FAKE_USERS_KWARGS = {
    'user__email__endswith': '.example.com',
    'user__email__endswith': '@example.com',
}
# ...
# inside our methods...
query = queryset.exclude(**EXCLUDE_FAKE_USERS_KWARGS)
```